string operation
string arm
BaxterArmJoints joints
geometry_msgs/Pose pick_pose
geometry_msgs/Pose[] waypoints
geometry_msgs/Pose place_pose
---
string operation
string arm
moveit_msgs/RobotTrajectory[] trajectories
